micro-rdk-ffi/example/example.c

```diff
@@ -24,9 +25,38 @@ int config_my_generic_sensor_A(struct config_context *ctx, void *user_data,
   struct my_generic_sensor_A *sensorA = malloc(sizeof(struct my_generic_sensor_A));
 
   sensorA->an_int = 1234567;
-  sensorA->array =  NULL;
   sensorA->an_int_from_config = my_int;
-  
+  int32_t vec_len = -1;
+  viam_code ret2 = config_get_i32_vec_len(ctx, "my_list", &vec_len);
+  if (ret2 != VIAM_OK) {
```

Comment:
If we can sanitize that code a bit so it's more readable that would be great!!
can return early when running into an error

Reply:
I adjusted it and cleaned it up a little, was it what you had in mind?